js/build-rss-feed.js

```diff
@@ -192,59 +211,67 @@ var rss = (function() {
         removeUnusedFeedsFromPV(feedsInUse);
       }, function() {
         alert('Error message here', function() {}, 'Error', 'Close');
+
         return false;
       });
     });
   }
 
   /**
    * removes feeds fom Persistence variables that are not in use.
-   * @param feedsInUse feeds in the configuration.
+   * @param {String[]} feedsInUse - feeds in the configuration.
+   * @return {void}
    */
   function removeUnusedFeedsFromPV(feedsInUse) {
     var inUse = false;
+
     for (var j = 0; j < window.pvObj.feeds; j++) {
       for (var w = 0; w < feedsInUse.length; w++) {
         if (feedsInUse[j] === window.pvObj.feeds[w].feed.uniqueName) {
           inUse = true;
         }
       }
+
       window.pvObj.feeds.remove(w);
     }
   }
+
   /**
    * based on a unique name get the feed Persistence variable with that uniqueName
-   * @param feedUniqueName feed unique name
-   * @param rssPVs persistence variable with feeds available
-   * @returns {*}
```

Comment:
Although `return` is a valid synonim, `returns` is the preferred tag to use https://jsdoc.app/tags-returns.html

Reply:
Thanks. I'll use `@returns` from now on.